README.md

```diff
@@ -1,5 +1,5 @@
-META-PELUX
-==========
+# meta-pelux
+
```

Comment:
I much prefer the preset style, using ==== dividers over #
Makes it easier to read in a text editor :)